lib/http.js

```diff
@@ -450,6 +450,28 @@ async function _authorize({
   };
 }
 
+/**
+ * Gets an expectedRootCapability from an invocationTarget.
+ *
+ * @param {object} options - Options to use.
+ * @param {string} options.invocationTarget - An invocationTarget.
+ * @param {string} options.keystoreId - A url to a key store.
+ *
+ * @return {string} The expectedRootCapability for the invocationTarget.
+ */
+function _getExpectedRootCapability({invocationTarget = '', keystoreId}) {
+  // return the rootCapability for authorizations
+  if(invocationTarget.endsWith('authorizations')) {
+    return `${keystoreId}/zcaps/authorizations`;
+  }
+  // return the rootCapability for new key generation
+  if(invocationTarget.endsWith('keys')) {
+    return `${keystoreId}/zcaps/keys`;
+  }
+  // this will succeed if the target is key
```

Comment:
Looks to me like the default of `''` passes successfully to this point which doesn't seem good.  I think it's best to use the `assert-plus` lib as elsewehre and ensure that it's a string at the top of the function.

Reply:
assert-plus is not used in this module at all, but yeah undefined is an invocationTarget should probably fail. Made the change here: 

Also an array is a valid invocationTarget so I added a check for that. We don't have any expectedRootCapabilities in this module that are arrays so I just return the invocationTarget (and possibly let it fail elsewhere) if the invocationTarget is an array.

https://github.com/digitalbazaar/bedrock-kms-http/pull/14/commits/7383616043e23e4795b4be031b6a51b66f5ad0d8